content/scalability/docs/control-plane.md

```diff
@@ -0,0 +1,164 @@
+# Kubernetes Control Plane
+
+The Kubernetes control plane is the Kubernetes components such as the API Server, kubelet, and Kubernetes Controller Manager. Scalability limits of these components are different depending on what you’re running in the cluster, but the areas with the biggest impact to scale include the Kubernetes version, utilization, and individual Node scaling.
```

Comment:
Similar as comment earlier - was surprised to see "kubelet" here. also, perhaps we should add "etcd" and "kube scheduler" for completeness. https://kubernetes.io/docs/concepts/overview/components/